integration/local/index.mjs

```diff
@@ -0,0 +1,25 @@
+import puppeteer from "puppeteer";
+import dappeteer from "dappeteer";
+import { transferHolder } from "./transfer-holder.mjs";
+import { surrender } from "./surrender.mjs";
+
+const main = async () => {
+  const browser = await dappeteer.launch(puppeteer, {
+    headless: false, // https://github.com/puppeteer/puppeteer#default-runtime-settings
```

Comment:
Why is it being run non-headless?

Reply:
it got stuck if headless:

<img width="1056" alt="Screenshot 2020-10-20 at 9 43 54 AM" src="https://user-images.githubusercontent.com/4774314/96529930-e6c6c200-12b8-11eb-9003-3bd9fc93f258.png">

https://github.com/TradeTrust/tradetrust-website/pull/253/checks?check_run_id=1278499255

im guessing this: 
- https://stackoverflow.com/questions/45372066/is-it-possible-to-run-google-chrome-in-headless-mode-with-extensions#:~:text=No%2C%20google%2Dchrome%20in%20headless,doesn't%20supports%20extensions%20yet.
- https://github.com/puppeteer/puppeteer/issues/659